README.md

```diff
@@ -836,6 +836,74 @@ utils.distributed_cmd_run(train)
 </p>
 </details>
 
+<details>
+<summary>CV - MNIST with Metric Learning</summary>
+<p>
+
+```python
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+
+from catalyst import contrib as ctb, data, dl
+from catalyst.core.callbacks import ControlFlowCallback
+from catalyst.dl.callbacks.metrics.cmc import CMCScoreCallback
```

Comment:
you should have access to them through `dl.{CallbackName}`

Reply:
yes, but this block of code became ugly after using dl.bla-bla
```
# 4. training with catalyst Runner
callbacks = [
    ControlFlowCallback(dl.CriterionCallback(), loaders="train"),
    ControlFlowCallback(CMCScoreCallback(topk_args=[1]), loaders="valid"),
    dl.callbacks.PeriodicLoaderCallback(valid=600),
]
```
Is it okay?

Reply:
this one looks worse, imho 
```
    # 4. training with catalyst Runner
    callbacks = [
        ControlFlowCallback(dl.CriterionCallback(), loaders="train"),
        ControlFlowCallback(
            dl.callbacks.metrics.cmc.CMCScoreCallback(topk_args=[1]),
            loaders="valid",
        ),
        dl.callbacks.PeriodicLoaderCallback(valid=600),
    ]
```

Reply:
it should be
```python
callbacks = [
        dl.ControlFlowCallback(dl.CriterionCallback(), loaders="train"),
        dl.ControlFlowCallback(dl.CMCScoreCallback(topk_args=[1]),loaders="valid"),
        dl.PeriodicLoaderCallback(valid=600),
    ]
```

Reply:
ah, ok, will try

Reply:
done